### DISCUSSION TITLE
Safari keyboard navigation does not work

### DISCUSSION THREAD

**Discussion by sk-roofr** [gpbl/react-day-picker#1754] [Support] [answered]
### Bug description

When I open the demo page in Safari, the keyboard navigation does not work.

MacOs: Ventura 

### To reproduce

Fork this CodeSandbox: https://codesandbox.io/s/react-day-picker-8-vl4ry with the code to reproduce the issue.

It does not work with any keyboard navigation.

### Expected behavior
Arrow and date get highlighted when the user presses the tab.

**Comment by gpbl** [comment]
@sk-roofr I cannot reproduce with safari 16.4.

**Reply by sk-roofr** [reply, marked as answer]
Actually I needed to enable the setting for the safari 
https://www.tempertemper.net/blog/how-to-use-the-keyboard-to-navigate-on-safari
